torchgeo/trainers/cyclone.py

```diff
@@ -222,7 +237,7 @@ def train_dataloader(self) -> DataLoader[Any]:
             batch_size=self.batch_size,
             num_workers=self.num_workers,
             shuffle=True,
-            pin_memory=True,
+            pin_memory=False,
```

Comment:
I believe False is the default, so we could just remove this argument.

Reply:
Done